### PR TITLE
Calendar view

### PR DESCRIPTION
**Pull request by annatraussnig**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,15 +1,25 @@
 import React from 'react'
 import { withRouter } from 'next/router'
 import { compose } from 'react-apollo'
+import { css } from 'glamor'
 
-import withAuthorization from '../components/Auth/withAuthorization'
+import { linkRule } from '@project-r/styleguide'
```
Comment (ovbm) [inline pages/index.js:6]: linkRule is deprecated and will be removed once darkmode is fully implemented, use `import { A } from '@project-r/styleguide` instead

File: components/Repo/Phases.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { css, merge } from 'glamor'
+import { Link } from '../../lib/routes'
+import { withRouter } from 'next/router'
+import { fontStyles } from '@project-r/styleguide'
+
+const styles = {
+  phase: css({
+    color: '#fff',
+    padding: '5px 10px',
```
Comment (ovbm) [inline components/Repo/Phases.js:10]: I actually prefer the padding of the current Phase component: '3px 6px' a bit more, makes it a bit more condensed.

File: components/Calendar/Day.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  const [colorScheme] = useColorContext()
+  const reposByTemplate = group(repos, repo =>
+    repo.latestCommit.document.meta.template === 'editorialNewsletter'
+      ? 'newsletter'
+      : 'other'
+  )
+  return (
+    <div
+      {...styles.container}
+      {...colorScheme.set('color', isPast ? 'textSoft' : 'text')}
```
Comment (ovbm) [inline components/Calendar/Day.js:71]: question: wouldn't it be easier to just set an opacity: 0.6 (or so) on the entire day? Also, it seems like in the current version, dates are still changeable on past repos, is that intentional?

File: components/Repo/Row.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+import { compose, graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import GithubIcon from 'react-icons/lib/fa/github'
+import LockIcon from 'react-icons/lib/md/lock'
+import PublicIcon from 'react-icons/lib/md/public'
+import { renderMdast } from 'mdast-react-render'
+import { matchType } from 'mdast-react-render/lib/utils'
+
+import { linkRule, A, Label, colors } from '@project-r/styleguide'
```
Comment (ovbm) [inline components/Repo/Row.js:11]: linkrule

File: components/Calendar/Day.js (hunk 3 above)
Comment (annatraussnig) [reply components/Calendar/Day.js:71]: I believe things can be republished – if you add a correcting for instance or if you make edits to some pages. Can check with production though.

**Review by ovbm — APPROVED** [review]

File: components/Repo/Add.js [hunk 5]
```diff
@@ -150,7 +155,7 @@ const TemplatePicker = compose(
         onChange({
           schema: item.value,
           templateRepoId: undefined,
-          templatePrefix: undefined
+          specialPrefix: undefined
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply components/Repo/Add.js:158]: When a repo is generated from a template, we use the id of the template minus the template prefix as prefix for the id of the newly generated repo.
So if the template is `template-www`, a derived repo would be `www-my-repo`.
`templatePrefix` denotes the `template` part in the template repo id.
`specialPrefix`here would be `www`. 
I'll try and come up with something… Any naming suggestions on your side?

File: components/Calendar/Repo.js [hunk 6]
```diff
@@ -0,0 +1,180 @@
+import React from 'react'
+import { css } from 'glamor'
+import { Link } from '../../lib/routes'
+import {
+  fontStyles,
+  useColorContext,
+  inQuotes,
+  colors
+} from '@project-r/styleguide'
+import { getLabel, getTitle } from '../Repo/utils'
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply components/Calendar/Repo.js:10]: Yeah, but from another folder. I will merge them and put everything in `lib/utils`

**Review by ovbm — APPROVED** [review]

File: components/Calendar/config.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  ],
+  articles: [
+    {
+      publicationDays: [3],
+      publicationTime: '4:30',
+      repoId: `${GITHUB_ORG}/template-am`
+    }
+  ]
+}
```
Comment (annatraussnig) [inline components/Calendar/config.js:28]: @patrickvenetz would you mind considering if there is a better solution for this? `repoId` refers to template repo ids and shouldn't be the same in staging and production.

**Comment by ovbm** [comment]
🥳

**Pull request merged**